docs/manual/replacements_and_deletions.md

```diff
@@ -27,7 +27,7 @@ The following changes can only be rolled out through replacement:
 The number of inflight replacements can be configured by setting `maxConcurrentReplacements`, per default the operator will replace all misconfigured process groups.
 Depending on the cluster size this can require a quota that is has double the capacity of the actual required resources.
 
-## Automatic Replacements
+## Automatic Replacements for Pods in Bad State
```

Comment:
```suggestion
## Automatic Replacements for ProcessGroups in Bad State
```

Please use the correct term in the documentation, the operator acts on ProcessGroups and the Pods are only one part of it. What is the benefit of adding `for ProcessGroups in Bad State`? What else should the operator automatically be replacing?

Reply:
It's to distinguish from Replacements for Pods on Tainted Nodes.